### PR TITLE
Concurrent bid requests

### PR DESCRIPTION
**Pull request by protonate**
@mkendall07 for discussion

### CODE REVIEW

File: .jscsrc [hunk 1]
```diff
@@ -7,5 +7,6 @@
   "validateIndentation": 2,
   "disallowSpacesInFunctionDeclaration": {
     "beforeOpeningRoundBrace": true
-  }
+  },
+  "maximumLineLength": 160
```
Comment (mkendall07) [inline .jscsrc]: 👍

File: package.json [hunk 2]
```diff
@@ -11,19 +11,11 @@
     "url" : "https://github.com/prebid/Prebid.js.git"
   },
   "adapters": [
```
Comment (protonate) [inline package.json]: @mkendall07 with this commit and by removing bidders here we can demo rendering ads as soon as bids are back. With all bidders the setTimeout always is reached due to testing config differences between bidders.

File: src/bidmanager.js [hunk 3]
```diff
@@ -352,84 +243,17 @@ function triggerAdUnitCallbacks(adUnitCode) {
...
 }
 
-function processCallbacks(callbackQueue, params) {
+function processCallbacks(callbackQueue) {
   var i;
   if (utils.isArray(callbackQueue)) {
     for (i = 0; i < callbackQueue.length; i++) {
       var func = callbackQueue[i];
-      callFunction(func, params);
+      func.call(pbjs, pbjs._bidsReceived);
```
Comment (protonate) [inline src/bidmanager.js]: wrap this in try/catch

**Comment by protonate** [comment]
Closing in favor of #353

**Pull request closed without merging**